docs/GE_Checklist.md

```diff
@@ -6,40 +6,138 @@ to be taken in order to publish a new FIWARE GE.
 Any doubt or request to the FIWARE Foundation shall be made by sending an email
 to [fiware-help@lists.fiware.org](mailto:fiware-help@lists.fiware.org)
 
--   New entry in the [FIWARE Catalogue](https://catalogue-server.fiware.org)
-    under the corresponding Chapter agreed at TSC Level. See also
+### New Generic Enablers
+
+The following mimimal requirements must be met before a new contribution can be
+considered as an Incubated Generic Enabler.
+
+-   All New Generic Enablers **MUST** be presented to the TSC for candidature
+    and accepted into an existing Chapter
```

Comment:
the acceptance can take time, for instance to allow studying the quality of the product, internal architecture, software design, code cleanliness etc.

Reply:
Fixed 866fc44d1 - added acceptance text